ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml

```diff
@@ -3059,10 +3066,19 @@
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Test Case</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>

```

Comment:
it should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments

Reply:
or like MS-OXWSCORE_R1732

Reply:
<ns1:REQ_ID>MS-OXWSCORE_R1730</ns1:REQ_ID>
....
<ns1:VerificationComment>Verified by derived requirement: MS-OXWSCORE_R1731.</ns1:VerificationComment>